### PR TITLE
Roulette fix

### PR DESCRIPTION
**Pull request by Vitiation**

### CODE REVIEW

**Review by DanRJ — APPROVED** [review]

File: crazybot/cogs/bj_game/blackjack.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        dealer_hand = [self.draw_card(), self.draw_card()]
+
+        # Show initial hands
+        await self.show_hand(ctx, player_hand, ctx.author.display_name)
+        initial_message+=f"Dealer's hand: {dealer_hand[0][0]}{dealer_hand[0][1]} ?"
+
+        # Add reactions for "Hit" and "Stand"
+        initial_message +="React with 🃏 to Hit or ✋ to Stand!"
+        message = await ctx.send(initial_message)
+        print(message)
```
Comment (MartinRovang) [inline crazybot/cogs/bj_game/blackjack.py:76]: remove

File: crazybot/cogs/game/slot_machine.py [hunk 2]
```diff
@@ -222,7 +235,7 @@ def generate_paylines(self, num_rows, num_cols):
             paylines["Diagonal Bottom-Left to Top-Right"] = [(i, num_cols - 1 - i) for i in range(num_rows)]
 
         return paylines
-
+    """"
```
Comment (MartinRovang) [inline crazybot/cogs/game/slot_machine.py:238]: bare 3 fnutter

File: crazybot/config/config.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 # Database settings
 
-DB_NAME =  '/data/coins.db'
+DB_NAME =  'C:/Users/Moh/Documents/GitHub/CrazyBlazin/data/coins.db'
```
Comment (MartinRovang) [inline crazybot/config/config.py:17]: gay

**Review by MartinRovang — CHANGES_REQUESTED** [review]

**Review by MartinRovang — CHANGES_REQUESTED** [review]
vbump v1.6.6

**Review by MartinRovang — APPROVED** [review]

**Pull request merged**